extensions/amp-analytics/0.1/events.js

```diff
@@ -389,6 +395,43 @@ export class VisibilityTracker extends EventTracker {
   }
 
   /**
+   * @param {string=} waitForSpec
```

Comment:
`=` means optional. This is non-optional param, although `?` and `|undefined` are allowed values. Ditto `selector`. `element` however is indeed optional.